frontend/src/Commons/Forms/FormModule/FormModule.tsx

```diff
@@ -0,0 +1,46 @@
+/**
+ * @summary A Form Section, container for an Input Module
+ * @desc A Form Module is a component that takes a form section container,
+ *       containing a header and an Input Module
+ *
+ * @param InputModule: ComponentType<string> => Takes string name of a Component, InputModule
+ * @param name: string => name of the Form Section
+ * @param enable: boolean => expand or collapse form section upon render
+ * @param faIcon: IconDefinition => Font Awesome IconDefinition to be passed to FormHeader component
+ *
+ * @example:
+      <FormModule
+        InputModule={farmInformation}
+        name="Fields and Soil"
+        faIcon={faWheatAwn}
+        enable={false}
+      />
+ *
+ * @author @GDamaso
+ */
+import React, { useState } from 'react';
+import InputModuleInterface from 'src/Interface/InputModuleinterface.tsx';
+import FormHeader from '../FormHeader/FormHeader.tsx';
+import { StyledFormContainer, StyledFormContent } from './FormModule.style';
+
+interface FormModuleProps {
+  InputModule: InputModuleInterface;
+}
+
+const FormModule: React.FC<FormModuleProps> = ({ InputModule }) => {
+  const { InputModuleComponent, name, enable, faIcon } = InputModule;
+  const [active, setActive] = useState(enable);
+
+  return (
+    <StyledFormContainer>
+      <FormHeader text={name} active={active} setActive={setActive} faIcon={faIcon} />
+      {active ? (
+        <StyledFormContent>
+          <InputModuleComponent />
+        </StyledFormContent>
+      ) : null}
```

Comment:
Pretty sure you can do `active && ('components')` instead of creating a null ternary

Reply:
Yes. In some situations you need to be mindful that if the value before && would ever evaluate to another falsy value like 0 or an empty string ("") the component would not show.